hedera-node/src/main/java/com/hedera/services/legacy/core/KeyPairObj.java

```diff
@@ -38,15 +49,59 @@ public KeyPairObj(String publicKey, String privateKey) {
     this.privateKey = privateKey;
   }
 
-  public String getPublicKeyAbyteStr() throws InvalidKeySpecException {
-    return CommonUtils.hex(((EdDSAPublicKey)getPublicKey()).getAbyte());
+
```

Comment:
Shouldn't this class also have 4 spaces indentation instead of two?

Reply:
Fixed.